book/src/getting_started.md

```diff
@@ -0,0 +1,91 @@
+# Getting started with Ariel OS
+
```

Comment:
A short intro with the goal of this page (what to expect) and that only Linux is supported would be good to have here:

```suggestion
This guide provides instructions on how to get started with Ariel OS with the goal to have the 'hello world' example running on a supported board. It starts with installing the prerequisites for building the Ariel OS, followed by  the commands to build and flash the example. Following the guide should take between 5 - 15 minutes depending on the speed of the computer.

Currently only Linux is supported for this guide, with installation instructions specific for Ubuntu-based distributions. Other Linux-based distributions should also work when the necessary packages are installed.
```

As usual, feel free to modify